app/src/main/java/de/westnordost/streetcomplete/data/user/UserModule.kt

```diff
@@ -36,6 +37,12 @@ val OAUTH2_REQUIRED_SCOPES = listOf(
 
 val userModule = module {
 
+    if (USE_TEST_API) {
+        OAUTH2_TOKEN_URL = "https://master.apis.dev.openstreetmap.org/oauth2/token"
+        OAUTH2_AUTHORIZATION_URL = "https://master.apis.dev.openstreetmap.org/oauth2/authorize"
+        OAUTH2_CLIENT_ID = "ObZ7yPf4lfs4XJ3NWysI3ukJMN0SHey1oPnNQnLmvw8"
```

Comment:
@westnordost do you want own client_id for test API or is this fine?

Reply:
fine for me to not own it.